docs/txp.md

```diff
@@ -267,7 +282,7 @@ inputs, tx is considered invalid. Otherwise this check succeeds and
 the difference between sum of inputs and sum of outputs is considered
 to be transaction fee.
 
-#### Fee check
+##### Fee check
```

Comment:
When there are 5 levels of nesting, it's likely a good indicator that something should be changed :/